split lakectl cmds - part 2

nopcoder opened the pull request:
part2 - end result will be based on https://github.com/treeverse/lakeFS/pull/6254

N-o-Z commented on cmd/lakectl/cmd/tag.go:
```diff
@@ -146,15 +147,15 @@ var tagShowCmd = &cobra.Command{
 		client := getClient()
 		u := MustParseRefURI("tag", args[0])
-		Fmt("Tag ref: %s\n", u.String())
+		fmt.Printf("Tag ref: %s\n", u.String())
```
```suggestion
		fmt.Printf("Tag ref: %s\n", u)
```

N-o-Z commented on cmd/lakectl/cmd/tag.go:
```diff
@@ -130,7 +131,7 @@ var tagDeleteCmd = &cobra.Command{
 		client := getClient()
 		u := MustParseRefURI("tag", args[0])
-		Fmt("Tag ref: %s\n", u.String())
+		fmt.Printf("Tag ref: %s\n", u.String())
```
```suggestion
		fmt.Printf("Tag ref: %s\n", u)
```

N-o-Z commented on cmd/lakectl/cmd/ingest.go:
```diff
@@ -83,7 +84,7 @@ var ingestCmd = &cobra.Command{
 			err = walker.Walk(ctx, block.WalkOptions{}, func(e block.ObjectStoreEntry) error {
 				if dryRun {
-					Fmt("%s\n", e)
+					fmt.Printf("%s\n", e)
```
```suggestion
					fmt.Println("%s", e)
```

nopcoder replied:
can't use %s with Println - I've just passed e

N-o-Z commented on cmd/lakectl/cmd/diff.go:
```diff
@@ -54,15 +54,15 @@ var diffCmd = &cobra.Command{
 			// got one arg ref: uncommitted changes diff
 			branchURI := MustParseRefURI("ref", args[0])
-			Fmt("Ref: %s\n", branchURI.String())
+			fmt.Printf("Ref: %s\n", branchURI)
```
```suggestion
			fmt.Println("Ref: %s", branchURI)
```

nopcoder replied:
kept Printf to use %s - we can drop the formatting and switch to use Println, but it is pretty equal.

N-o-Z commented on cmd/lakectl/cmd/commit.go:
```diff
@@ -55,7 +56,7 @@ var commitCmd = &cobra.Command{
 
 		branchURI := MustParseRefURI("branch", args[0])
-		Fmt("Branch: %s\n", branchURI.String())
+		fmt.Printf("Branch: %s\n", branchURI)
```
```suggestion
		fmt.Println("Branch: %s", branchURI)
```

N-o-Z commented on cmd/lakectl/cmd/cherry_pick.go:
```diff
@@ -27,7 +28,7 @@ var cherryPick = &cobra.Command{
 		ref := MustParseRefURI("ref", args[0])
 		branch := MustParseBranchURI("branch", args[1])
-		Fmt("Branch: %s\n", branch.String())
+		fmt.Printf("Branch: %s\n", branch)
```
```suggestion
		fmt.Println("Branch: %s", branch)
```

nopcoder replied:
kept the format - can switch to use Println and drop the format but it is the same

N-o-Z commented on cmd/lakectl/cmd/cat_hook_output.go:
```diff
@@ -17,14 +18,14 @@ var catHookOutputCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		u := MustParseRepoURI("repository", args[0])
-		Fmt("Repository: %s\n", u.String())
+		fmt.Printf("Repository: %s\n", u)
```
```suggestion
		fmt.Println("Repository: %s", u)
```

nopcoder replied:
kept the format - can switch to use Println and drop the format but it is the same

N-o-Z commented on cmd/lakectl/cmd/cat_hook_output.go:
```diff
@@ -17,14 +18,14 @@ var catHookOutputCmd = &cobra.Command{
 		resp, err := client.GetRunHookOutputWithResponse(ctx, u.Repository, runID, hookRunID)
 		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
-		Fmt("%s\n", string(resp.Body))
+		fmt.Printf("%s\n", string(resp.Body))
```
```suggestion
		fmt.Println("%s", string(resp.Body))
```

nopcoder replied:
switched to use Println without %s (no format support)

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -68,7 +68,7 @@ var branchCreateCmd = &cobra.Command{
 			DieFmt("failed to parse source URI: %s", err)
 		}
-		Fmt("Source ref: %s\n", sourceURI.String())
+		fmt.Printf("Source ref: %s\n", sourceURI.String())
```
```suggestion
		fmt.Println("Source ref: %s", sourceURI)
```

nopcoder replied:
kept the original

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -78,7 +78,7 @@ var branchCreateCmd = &cobra.Command{
 		})
 		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusCreated)
-		Fmt("created branch '%s' %s\n", u.Ref, string(resp.Body))
+		fmt.Printf("created branch '%s' %s\n", u.Ref, string(resp.Body))
```
```suggestion
		fmt.Println("created branch '%s' %s", u.Ref, string(resp.Body))
```

nopcoder replied:
kept the original

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -95,7 +95,7 @@ var branchDeleteCmd = &cobra.Command{
 		client := getClient()
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```
```suggestion
		fmt.Println("Branch: %s", u)
```

nopcoder replied:
kept the original

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -116,7 +116,7 @@ var branchRevertCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```
```suggestion
		fmt.Println("Branch: %s\n", u)
```

nopcoder replied:
kept the format and dropped the call to String()

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -135,7 +135,7 @@ var branchRevertCmd = &cobra.Command{
 			})
 			DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusNoContent)
-			Fmt("commit %s successfully reverted\n", commitRef)
+			fmt.Printf("commit %s successfully reverted\n", commitRef)
```
```suggestion
			fmt.Println("commit %s successfully reverted", commitRef)
```

nopcoder replied:
kept the original

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -154,7 +154,7 @@ var branchResetCmd = &cobra.Command{
 		clt := getClient()
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```
```suggestion
		fmt.Println("Branch: %s", u)
```

nopcoder replied:
kept the original format and dropped the call to String()

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -205,14 +205,14 @@ var branchShowCmd = &cobra.Command{
 		client := getClient()
 		u := MustParseBranchURI("branch", args[0])
-		Fmt("Branch: %s\n", u.String())
+		fmt.Printf("Branch: %s\n", u.String())
```
```suggestion
		fmt.Println("Branch: %s", u)
```

nopcoder replied:
kept Printf and dropped the call to String()

N-o-Z commented on cmd/lakectl/cmd/branch.go:
```diff
@@ -205,14 +205,14 @@ var branchShowCmd = &cobra.Command{
 		}
 		branch := resp.JSON200
-		Fmt("Commit ID: %s\n", branch.CommitId)
+		fmt.Printf("Commit ID: %s\n", branch.CommitId)
```
```suggestion
		fmt.Println("Commit ID: %s", branch.CommitId)
```

nopcoder replied:
kept the original

N-o-Z reviewed (changes requested):
Partial review

nopcoder commented:
@N-o-Z addressed all the comments some of them are part of the next PR.
- This PR kept all `auth*` changes
- Create part2b (https://github.com/treeverse/lakeFS/pull/6285) with all the rest of the had in this PR.
- Part3 rebased on part2b

N-o-Z reviewed (changes requested):
@nopcoder Thanks!
One comment regarding file naming
